### PR TITLE
Release/1.1.3

### PR DESCRIPTION
**Pull request by ashleyfae**
Milestone: https://github.com/easydigitaldownloads/edd-add-to-cart-popup/milestone/1?closed=1

| Issue # | Description |
|---------|-------------|
| #8 | Link to HTTPS page version |
| #7 | Fix typo |
| #6 | Popup preview can be seen by any logged in user |
| #5 | Error after clicking "Preview Popup" button |
| #4 | Update plugin author and add updater code |
| #13 | 3.0 - EDD Licensing not working |

Note I accidentally merged this one into the main branch so it won't appear in the diff: https://github.com/easydigitaldownloads/edd-add-to-cart-popup/pull/8/files

Also note: It took me a while to track down that there's no constant for the version number. In licensing, the version number is passed in here: https://github.com/easydigitaldownloads/edd-add-to-cart-popup/blob/develop/includes/Aventura/Edd/AddToCartPopup/Core/Plugin.php#L80 via `$this->getInfo('Version')` That data comes from `_loadInfo()` which sets the info from the plugin headers: https://github.com/easydigitaldownloads/edd-add-to-cart-popup/blob/develop/includes/Aventura/Edd/AddToCartPopup/Core/Plugin.php#L252

tl;dr: Version number comes from the main plugin header.

### CODE REVIEW

**Review by robincornett — APPROVED** [review]
Willing to approve this but the item ID could still be added to the `EDD_License` call: `805899` is what it seems to be.

**Comment by ashleyfae** [comment]
@robincornett Added.

**Review by robincornett — COMMENTED** [review]
Sorry, checking this again and I don't think the conditional is triggering to instantiate the `EDD_License` call at all. It's not showing on the licensing screen in the Downloads > Settings, and no http call is being recorded for it, either. I'm guessing it needs to be hooked into `admin_init` maybe instead of being just in the constructor?

(Wondering if current users will see an update notification at all....)

**Comment by ashleyfae** [comment]
You're right. Looks like that's been broken for a while. I'll make a new issue.

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,11 @@
+= 1.1.3 (2021-July-21) =
+* Improvement: The "Preview Popup" button has been removed from the top admin bar. It wasn't working on any page other than the add-on's own settings page, and that page already has a dedicated button anyway.
```
Comment (robincornett) [inline changelog.txt:2]:
```suggestion
* Improvement: The "Preview Popup" button has been removed from the top admin bar. It wasn't working on any page other than the extension's own settings page, and that page already has a dedicated button anyway.
```
Minor detail but my understanding is that we should be using extension instead of add-on (support guidelines).

Comment (ashleyfae) [reply changelog.txt]: Hard habit to break!

**Review by robincornett — APPROVED** [review]
